### ISSUE TITLE
Split api endpoints into separate servers. For example, mailhook handlers versus user facing endpoints.

### ISSUE THREAD

**Issue by oli-kitty** [47deg/TAFutureWork#11]

**Comment by toddburnside** [comment]
### Monolithic API

#### Current State
There is a single, monolithic, Play API server that handles both user facing endpoints (ex. debtor making a payment) and endpoints handling automated traffic (such as webhooks). This requires a single server instance scaled to handle all of the incoming traffic of all types and does not allow scaling based on traffic type. 

#### Suggestion
Break the server into individual pieces based on type of traffic. More investigation would need to be done to determine exactly how to split it up, but some ideas are:
- endpoints used by debtors
- webhooks endpoints
- individual integrations
- Engage endpoints.

This is a subset of #3, and depends on #4
